### DISCUSSION TITLE
Windows Defender SmartScreen prevented an unrecognized app from starting. Running this app might put your pc at risk

### DISCUSSION THREAD

**Discussion by mubshrx** [microsoft/winget-pkgs#3335] [General]
# Brief description of your issue

I created a manifest file for [RunJS](https://github.com/lukehaas/RunJS) and to validate the manifest I tried to install RunJS with the newly created manifest file. I got this error:
**Windows Defender SmartScreen prevented an unrecognized app from starting. Running this app might put your PC at risk.**

Why do I get this issue and what is the solution?

**Comment by OfficialEsco** [comment]
https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-smartscreen/microsoft-defender-smartscreen-overview

There is no way to unblock it other than waiting for the Microsoft Team to clear it. For very good reasons. The dev need to sign the software to not go trough this process every time.
